src/graphql.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+const https = require('https');
+
+/** Escape strings to prevent injection attacks. Other types aren't an issue. */
+const escapeArgValue = val => {
+	if ('string' === typeof (val)) {
```

Comment:
Can be shortened: `return ('string' === typeof (val)) ? ... : val`

Reply:
It certainly can. Would that make it any better though?

Reply:
Nah.